library/build.gradle

```diff
@@ -59,4 +60,14 @@ dependencies {
     //Stetho
     implementation 'com.facebook.stetho:stetho:1.5.1'
     implementation 'com.facebook.stetho:stetho-okhttp3:1.5.1'
+
+    //Firebase
+    implementation 'com.google.firebase:firebase-messaging:20.1.0'
+
+    //Glide
+    implementation 'com.aminography:redirectglide:1.0.3'
+    implementation 'com.github.bumptech.glide:glide:4.9.0'
```

Comment:
Could look for a way to avoid using `Glide`? That was not a problem in UI SDK because it is used widely there already, but having dedicated library (with another extra library) just for notification icon loading is too much.

Reply:
@elevenetc We can use something like Picasso.

Reply:
That's the same. Any imaging library will occupy just half of the library. There should be way to do it with Android SDK classes.